openzaak/src/main/kotlin/com/ritense/openzaak/service/ZaakTypeService.kt

```diff
@@ -17,10 +17,13 @@
 package com.ritense.openzaak.service
 
 import com.ritense.openzaak.service.impl.model.ResultWrapper
+import com.ritense.openzaak.service.impl.model.catalogi.BesluitType
 import com.ritense.openzaak.service.impl.model.catalogi.ZaakType
 
 interface ZaakTypeService {
 
     fun getZaakTypes(): ResultWrapper<ZaakType>
 
+    fun getBesluitTypes(): ResultWrapper<BesluitType>
```

Comment:
Though i don't think it's bad to combine this, the class name doesn't match the method added.

Reply:
I moved it to a new client